dev/Interop/StoragePickers/PickerCommon.cpp

```diff
@@ -278,6 +341,28 @@ namespace PickerCommon {
         return result;
     }
 
+    void PickerParameters::CaptureFilterSpecData(
+        winrt::Windows::Foundation::Collections::IVectorView<winrt::hstring> fileTypeFilterView,
+        winrt::Windows::Foundation::Collections::IMapView<winrt::hstring, winrt::Windows::Foundation::Collections::IVector<winrt::hstring>> fileTypeChoicesView)
+    {
+        // The FileTypeChoices takes precedence over FileTypeFilter if both are provided.
+        if (fileTypeChoicesView && fileTypeChoicesView.Size() > 0)
+        {
+            CaptureFilterSpec(fileTypeChoicesView);
```

Comment:
>CaptureFilterSpec(fileTypeChoicesView); [](http://example.com/codeflow?start=12&length=39)

It looks like in this new case that `FocusLastFilter` does not get set to true so FileOpenPicker will not select a filter?